src/jit/codegenxarch.cpp

```diff
@@ -5329,13 +5331,13 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
     // Is the caller supposed to pop the arguments?
     if (fCallerPop && (stackArgBytes != 0))
     {
-        genAdjustSP(stackArgBytes);
+        stackAdjustBias = stackArgBytes;
```

Comment:
`genAdjustSP` seems to be for both Windows and Linux. But `genRemoveAlignmentAfterCall` works only for Linux.